pages/static/containers_js.go

```diff
@@ -28,7 +28,11 @@ function humanizeIEC(num) {
         var ret = humanize(num, 1024, ["TiB", "GiB", "MiB", "KiB", "Bytes"]);
 	return ret[0].toFixed(2) + " " + ret[1];
 }
-
+// IEC naming convention, with shorter abbreviations
```

Comment:
ok, changed the original one :) I couldn't find any place on the UI where the numbers are low enough to show up as bytes - other than the ps output.